plugins/plugin-debugger/che-plugin-debugger-ide/pom.xml

```diff
@@ -75,6 +75,10 @@
         </dependency>
         <dependency>
             <groupId>org.eclipse.che.plugin</groupId>
+            <artifactId>che-plugin-java-ext-lang-client</artifactId>
```

Comment:
Are you sure that we need dependency for `java` extension inside abstract debugger plugin?